app/views/registrations/edit.html.erb

```diff
@@ -1,5 +1,6 @@
-<% provide(:title, t('registrations.edit_registration.title', person: @user.person&.name || @user.name, comp: @competition.name)) %>
```

Comment:
Note for review: Whenever we link a `Person` back to a `User` (commonly known as "claim your WCA ID"), we also backfill the Person's name and DOB into the User.

In other words, claimed WCA IDs are always in sync between user and person automatically.

Reply:
Makes sense, but do we want to introduce an unrelated change here in this PR?

Reply:
It's not really an unrelated change, more of a fix for something that was clearly redundant before.
Also, it affects only the page title, i.e. the thing that gets displayed on your browser tab and the window frame, so if for some obscure reason this change messes something up, the impact is minimal.